scripts/runtime-upgrade/test_runtime_upgrade.sh

```diff
@@ -1,7 +1,24 @@
+#!/usr/bin/env bash
 
-block=$1
+set -euxo pipefail
 
-./target/release/zeitgeist export-state $block --chain battery_park --pruning archive > test-upgrade.json
-node scripts/runtime-upgrade/adjustTestUpgradeJson.js
-./target/release/zeitgeist purge-chain --chain ./test-upgrade.json -y
-./target/release/zeitgeist --chain ./test-upgrade.json --alice
\ No newline at end of file
+ALICE_PK="0x04d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d"
+BLOCK=$1
+KEY_TO_SEARCH="0x57f8dc2f5ab09467896f47300f0424385e0621c4869aa60c02be9adcc98a0d1d"
+PRODUCTION_IMAGE=zeitgeistpm/zeitgeist-node:latest
+
+# Sync with live network
+
+sudo docker run -d --name migration -v /tmp/migration:/zeitgeist/data $PRODUCTION_IMAGE --base-path /zeitgeist/data --chain battery_park --pruning archive
+sleep 30m
+sudo docker container stop migration
+sudo docker container rm migration
+cp -r /tmp/migration /tmp/migration-copy
+
+# Run new node with latest changes
+
+cargo build --bin zeitgeist --release
+./target/release/zeitgeist export-state --base-path /tmp/migration-copy --chain battery_park --pruning archive $BLOCK > /tmp/test-upgrade.json
+sed -i "s/\"$KEY_TO_SEARCH\": \".*\"/\"$KEY_TO_SEARCH\": \"$ALICE_PK\"/" /tmp/test-upgrade.json
+sed -i "s/\"0x5c0d1176a568c1f92944340dbfed9e9c530ebca703c85910e7164cb7d1c9e47b\": \".*\"/\"0x5c0d1176a568c1f92944340dbfed9e9c530ebca703c85910e7164cb7d1c9e47b\": \"0x04d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d\"/" /tmp/test-upgrade.json
```

Comment:
Can you make sure to also change the name, id, protocol id, and bootnodes so that this testing chain doesn't conflict with the actual Battery Park in any way?

Reply:
Indeed. Thanks!